core/src/test/java/com/sequenceiq/cloudbreak/service/stack/flow/UpdateNodeCountValidatorTest.java

```diff
@@ -189,9 +189,9 @@ public void testValidateStatusForStartHostGroup(
 
         if (errorMessageSegment.isPresent()) {
             checkExecutableThrowsException(errorMessageSegment, stackdto,
-                    () -> underTest.validateStackStatusForStartHostGroup(stackdto, instanceGroupAdjustmentV4Request));
+                    () -> underTest.validateStackStatusForStopStartHostGroup(stackdto, instanceGroupAdjustmentV4Request.getInstanceGroup()));
```

Comment:
Can a unit test be added to check if exceptions are thrown / not thrown when dependent services are unhealthy / healthy during downscale?

Reply:
Okay, I will add unit tests for downscale operation.